packages/ui/src/theme/tokens/components/storagemanager.ts

```diff
@@ -1,22 +1,172 @@
 import { DesignTokenProperties, OutputVariantKey } from '../types/designToken';
-import { fileuploader, FileUploaderTokens } from './fileUploader';
+import { TypographyTokens } from '../types/typography';
 
-export interface StorageManagerTokens<OutputType extends OutputVariantKey>
-  extends Omit<FileUploaderTokens<OutputType>, 'body' | 'previewer'> {
+type BaseDropZoneTokens<OutputType> = DesignTokenProperties<
+  | 'backgroundColor'
+  | 'borderWidth'
+  | 'borderStyle'
+  | 'borderColor'
+  | 'borderRadius',
+  OutputType
+>;
+export interface StorageManagerTokens<OutputType extends OutputVariantKey> {
+  dropzone?: DesignTokenProperties<
+    'gap' | 'paddingBlock' | 'paddingInline' | 'textAlign',
+    OutputType
+  > &
+    BaseDropZoneTokens<OutputType> & {
+      _active?: BaseDropZoneTokens<OutputType>;
+
+      icon?: DesignTokenProperties<'fontSize' | 'color', OutputType>;
+
+      text?: TypographyTokens<OutputType>;
+    };
+  file?: DesignTokenProperties<
+    | 'alignItems'
+    | 'backgroundColor'
+    | 'borderColor'
+    | 'borderRadius'
+    | 'borderStyle'
+    | 'borderWidth'
+    | 'gap'
+    | 'paddingBlock'
+    | 'paddingInline',
+    OutputType
+  > & {
+    name?: TypographyTokens<OutputType>;
+    size?: TypographyTokens<OutputType>;
+    image?: DesignTokenProperties<
+      'backgroundColor' | 'borderRadius' | 'color' | 'height' | 'width',
+      OutputType
+    >;
+  };
   filelist?: DesignTokenProperties<'flexDirection' | 'gap'>;
-  previewer?: {
+  loader?: DesignTokenProperties<
+    'strokeWidth' | 'strokeFilled' | 'strokeEmpty' | 'strokeLinecap',
+    OutputType
+  >;
+
+  previewer?: DesignTokenProperties<
+    | 'backgroundColor'
+    | 'borderColor'
+    | 'borderRadius'
+    | 'borderStyle'
+    | 'borderWidth'
+    | 'maxHeight'
+    | 'maxWidth'
+    | 'paddingBlock'
+    | 'paddingInline',
+    OutputType
+  > & {
+    text?: TypographyTokens<OutputType>;
+
+    body?: DesignTokenProperties<
+      'gap' | 'paddingInline' | 'paddingBlock',
+      OutputType
+    >;
     footer?: DesignTokenProperties<'justifyContent', OutputType>;
   };
 }
 
 export const storagemanager: Required<StorageManagerTokens<'default'>> = {
-  ...fileuploader,
```

Comment:
Removed tight coupling with fileuploader tokens and performed visual inspection to make sure styling was not impacted